node/src/request_coordinator.ts

```diff
@@ -11,16 +16,7 @@ export class RequestCoordinator {
     this.resolver = new DNS.Resolver();
   }
 
-  /**
-   * Send pause request to all hosts of a service records.
-   * If one of the hosts cannot be paused, this function unpause (rollback) all hosts.
-   * @param {boolean} waitOutstanding
-   * @param {number} maxPauseWaitTime
-   */
-  async pause(
-    waitOutstanding: boolean,
-    maxPauseWaitTime: number | null = null
-  ): Promise<void> {
+  async resolveClients(): Promise<AdminClient[]> {
```

Comment:
This is a small refactoring.
Since the pause, unpause and checkPause functions are using the same logic, it's better to be abstracted.